lib/puppet/functions/ldapquery.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# @summary DEPRECATED.  Use the namespaced function [`ldapquery::query`](#ldapqueryquery) instead.
```

Comment:
the next release is a major one. shouldn't we just drop it now?

Reply:
It's always nice to users to give them a release where something is deprecated before removing IMO.  It's not a lot of hassle for us to keep this for a release or two.

Reply:
yes. But we will also forget about it. After we merge this, can you create a new PR to drop this? so we keep it in mind.